ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/wizard/v1/wizard/clientlibs/site/css/wizard.css

```diff
@@ -55,10 +55,10 @@
 .cmp-adaptiveform-wizard__previousNav{
 
 }
-.cmp-adaptiveform-wizard__previousNav__hidden{
+.cmp-adaptiveform-wizard__previousNav--hidden{
     display: none;
 }
-.cmp-adaptiveform-wizard__nextNav__hidden{
+.cmp-adaptiveform-wizard__nextNav--hidden{
```

Comment:
This may introduce a backward-incompatible change; however, I believe the wizard component has not been customized by anyone yet. To minimize technical debt, I am approving the PR.